basics/indexing/forward-index.md

```diff
@@ -31,10 +45,24 @@ When a column is physically sorted, Pinot uses a sorted forward index with run-l
 
 The Sorted forward index has the advantages of both good compression and data locality. The Sorted forward index can also be used as an inverted index.
 
-### Real-time tables
+Sorted forward index are active when:
+* The segment is sorted by this column
```

Comment:
by what column?

Reply:
Indexes are applied column by column. So sorted forward index will be applied for any column `c` if the table is sorted by that column `c`.